model/ftn/wmesmfmd.ftn

```diff
@@ -534,6 +554,20 @@
       if (ESMF_LogFoundError(rc, PASSTHRU)) return
 !
 ! -------------------------------------------------------------------- /
+! Check if coupled with CMEPS mediator or not
```

Comment:
Is this just to check if the CMEPS mediator exists or if any mediator exists?

Reply:
This check is just for keeping CMEPS modifications as isolated. By this way, WW3 cap can be used with/without CMPES and we could keep continue to support NUOPC connector type coupling.

Reply:
I know this was tested and updated for the CMEPS mediator specifically, but is this checking to see if there is any mediator or just the CMEPS mediator?  The comment implies to me that it's the CMEPS mediator but the code seems to be general for any mediator.

Reply:
It is just for CMEPS. This specific attribute was already used by CMEPS and I decided to use it to follow the conventions.